backend/src/services/database/database-migration.service.ts

```
@@ -146,4 +150,113 @@ export class DatabaseMigrationService {
+          level: 'DANGER',
+          statement,
+        });
+        riskLevel = 'DANGER';
+      } else if (
+        upperStmt.includes('DROP COLUMN') ||
+        upperStmt.includes('DROP CONSTRAINT') ||
+        (upperStmt.includes('ALTER TABLE') && upperStmt.includes('DROP'))
+      ) {
+        riskFactors.push({
+          code: 'SCHEMA_BREAKING_CHANGE',
+          description: 'Statement drops columns or constraints which may break existing functionality.',
```

ESLint

[error] Insert `⏎···········`

(prettier/prettier)